server/api/projects/campaigns.py

```diff
@@ -125,57 +125,7 @@ def delete(self, project_id, campaign_id):
                 description: Internal Server Error
         """
         try:
-            new_campaigns = CampaignService.delete_project_campaign(
-                project_id, campaign_id
-            )
-            return new_campaigns.to_primitive(), 200
-        except NotFound:
-            return {"Error": "Campaign Not Found"}, 404
-        except Exception as e:
-            error_msg = f"Project Campaigns GET - unhandled error: {str(e)}"
-            current_app.logger.critical(error_msg)
-            return {"error": error_msg}, 500
-
-
-class ProjectsCampaignsActionsRemoveAPI(Resource):
-    @token_auth.login_required
-    def post(self, project_id):
-        """
-        Unassign campaign(s) for a project
-        ---
-        tags:
-          - campaign
```

Comment:
@xamanu what do you think about renaming the tag to project campaigns or just campaigns to have it together with the other endpoints on swagger?